create_teacher.rb

```diff
@@ -11,5 +12,6 @@ def create_teacher
 
   teacher = Teacher.new(age, specialization, name)
   @people.push(teacher)
+  save_teacher(age, specialization, name)
   puts 'Teacher is created successfully'
```

Comment:
- [ ] when creating a teacher the user receives the same error as for the student:
![afbeelding](https://user-images.githubusercontent.com/39240820/218078355-c999b958-a227-464f-9e81-35534a5705cb.png)